src/HTML5/Parser/Tokenizer.php

```diff
@@ -83,9 +83,9 @@ public function __construct($scanner, $eventHandler, $mode = self::CONFORMANT_HT
      */
     public function parse()
     {
-        $p = 0;
+        //$p = 0;
         do {
-            $p = $this->scanner->position();
+            //$p = $this->scanner->position();
```

Comment:
I wonder why we get the position and don't use it? In any case, these can likely be removed rather than commented out.